replace experimental hasVirtual/hasReal with getApplicationMode #437 (#438)
enonic/doc-xp: docs/api/lib-app.adoc

});
```

image:xp-7110.svg[XP 7.11.0,opts=inline]

=== getApplicationMode

> Fetches application mode of an application with the specified key.

Reviewer: ```suggestion
Fetches mode of an application with the specified key.
```

[.lead]
Returns

> *string* : Application mode. "bundled" - only installed and active application is presented. "virtual" - only virtual, a node-based application is presented. "augmented" - both bundled and virtual application are presented with passed application key.

Reviewer: ```suggestion
*string* : Application mode. Can be one of the following types:
* `bundled` - an installed and active application, no virtual app with the same key exists;
* 'virtual' - a "virtual", node-based application, no bundled app with the same key exists;
* 'augmented' - both bundled and virtual application co-exist with the same application key.
```

> NOTE: Installed but disabled bundled applications are considered missing.

Reviewer 2: ```suggestion
NOTE: Installed but disabled bundled applications are considered missing.

WARN: This is an experimental API.
```

[.lead]
Example

> .fetches application mode of an app with `my-app-key` key:

Reviewer: ```suggestion
.fetches mode of an application with `my-app-key` key:
```
